### PR TITLE
Opta Force-Unlock

### PR DESCRIPTION
**Pull request by nsarupr**
### Usage

> `opta force-unlock [-f/--force-terraform] [-c/--config <config-file>] [-e/--env]`

### Feature

> Gets the **Terraform out of the Locked State**, and also **Rolls back / Delete (Only for the revision==1) for helm in the pending-upgrade state.**

### Caution

> Please use this feature very carefully as it does not distinguish the Locked Terraform State where the changes are being performed presently, or if the state is actually stuck.

### CODE REVIEW

File: opta/commands/helm_rollback.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from opta.utils import check_opta_file_exists
+
+
+@click.command()
+@click.option("-c", "--config", default="opta.yml", help="Opta config file.")
+@click.option(
+    "-e", "--env", default=None, help="The env to use when loading the config file."
+)
+@click.option("-h", "--helm-release", default=None, help="The release to roll back.")
+def helm_rollback(config: str, env: Optional[str], helm_release: str) -> None:
```
Comment (juandiegopalomino) [inline opta/commands/helm_rollback.py:18]: delete if current release is 1

**Review by juandiegopalomino — COMMENTED** [review]
take care of the lint issue, add tests for the aws/gcp/azure get_terraform_lock_id to get over the 70% code coverage and looks great!

**Comment by nsarupr** [comment]
> take care of the lint issue, add tests for the aws/gcp/azure get_terraform_lock_id to get over the 70% code coverage and looks great!

@juandiegopalomino Lint issues don't seem to be popping up on local. 
As for the test cases, I am yet to add test cases for getting terraform lock id for various clouds.

File: opta/commands/destroy.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def destroy(config: str, env: Optional[str], auto_approve: bool) -> None:
         tf_flags.append("-auto-approve")
 
     gen_all(layer)
-    Terraform.init("-reconfigure")
+    Terraform.init(False, "-reconfigure")
```
Comment (nsarupr) [inline opta/commands/destroy.py:69]: @juandiegopalomino 
I have added the default value here. Let me know if you want to ignore the error instead.

**Review by juandiegopalomino — APPROVED** [review]
lgtm

**Pull request merged**